.github/workflows/release.yml

```diff
@@ -157,9 +115,23 @@ jobs:
         with:
           version: latest
 
-      - name: docker_build
+      - name: docker_build_and_push_version
+        run: |
+          docker buildx build --push \
+            --tag serowy/docker2mqtt:${{ steps.version_tag.outputs.version }}-${{ matrix.base_image }} \
+            --platform ${{ platform }} \
+            --file Dockerfile \
+            --build-arg BASE_IMAGE=${{ matrix.base_image }}:${{ matrix.version }} \
+            --build-arg SUB_DIR=${{ matrix.arch_rust }}-unknown-linux-${{ matrix.libc }}/ \
+            .
+
+      - name: docker_build_and_push_latest
+        if: ${{ matrix.tag_as_latest }}
         run: |
           docker buildx build --push \
-            --tag serowy/docker2mqtt:${{ steps.version_tag.outputs.version }} \
             --tag serowy/docker2mqtt:latest \
```

Comment:
We can push multiple tags with one build.

Reply:
We have to separate it as we can tag Debian and Alpine both as latest.

Reply:
Latest-alpine
Latest-...

Should be sufficient?

Reply:
Hmm, would be possible, but latest without suffix is a Docker standard for the default image. Would not drop it for convention reasons.